catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/CatalogDorisIT.java

```diff
@@ -185,20 +184,16 @@ void testDorisSchemaBasicOperation() {
     SupportsSchemas schemas = catalog.asSchemas();
 
     // test list schemas
-    NameIdentifier[] nameIdentifiers = schemas.listSchemas();
-    Set<String> schemaNames =
-        Arrays.stream(nameIdentifiers).map(NameIdentifier::name).collect(Collectors.toSet());
-    Assertions.assertTrue(schemaNames.contains(schemaName));
+    String[] schemaNames = schemas.listSchemas();
+    Assertions.assertTrue(Arrays.asList(schemaNames).contains(schemaName));
```

Comment:
Maybe you can check if you can use `ArrayUtils.contains()` without introducing extra dependencies.

Reply:
I don't understand; Arrays is the JDK utility class, while ArrayUtils is apache commons lang; As this is so simple a case, I don't think it is necessary to use ArrayUtils here.

Reply:
This is a personal preference and the current implementation looks good.

Reply:
ok, thanks~